src/main/java/org/openstreetmap/atlas/checks/validation/intersections/OceanBleedingCheck.java

```diff
@@ -37,15 +45,17 @@
             + "|wetland->bog,fen,mangrove,marsh,saltern,saltmarsh,string_bog,swamp,wet_meadow"
             + "|landuse->*";
     private final TaggableFilter invalidOceanTags;
+    private static final String DEFAULT_OCEAN_BOUNDARY_TAGS = "natural->coastline";
+    private final TaggableFilter oceanBoundaryTags;
     private static final String DEFAULT_OFFENDING_MISCELLANEOUS_LINEITEMS = "railway->rail,narrow_gauge,preserved,subway,disused,monorail,tram,light_rail,funicular,construction,miniature";
     private final TaggableFilter defaultOffendingLineitems;
     private static final String DEFAULT_HIGHWAY_MINIMUM = "TOLL_GANTRY";
     private final HighwayTag highwayMinimum;
     private static final List<String> DEFAULT_HIGHWAYS_EXCLUDE = Collections.emptyList();
     private final List<HighwayTag> highwaysExclude;
-    private static final String OCEAN_INSTRUCTION = "Ocean feature {0,number,#} has invalid intersections.";
-    private static final String BLEEDING_BUILDING_INSTRUCTION = "Building {0,number,#} intersects the ocean feature.";
-    private static final String BLEEDING_LINEITEM_INSTRUCTION = "Way {0,number,#} intersects the ocean feature.";
+    private static final String OCEAN_INSTRUCTION = "Ocean feature {0,number,#} has invalid intersections. ";
+    private static final String BLEEDING_BUILDING_INSTRUCTION = "Building {0,number,#} intersects the ocean feature. ";
+    private static final String BLEEDING_LINEITEM_INSTRUCTION = "Way {0,number,#} intersects the ocean feature. ";
```

Comment:
Is it possible to concatenate the ids into a single "Way" instruction? Same for "Building".
That way we can avoid instructions like:

Ocean feature 693833861 has invalid intersections.Way 781473903 intersects the ocean feature.Way 781473903 intersects the ocean feature.Way 781473903 intersects the ocean feature.Way 781473903 intersects the ocean feature.Way 373430804 intersects the ocean feature.Way 373430804 intersects the ocean feature.Way 373430800 intersects the ocean feature.Way 373430800 intersects the ocean feature.